openage/convert/processor/export/media_exporter.py

```diff
@@ -5,6 +5,7 @@
 Converts media requested by export requests to files.
 """
 import os
+import logging
```

Comment:
openage has its own logging implementation in `openage/log`. You can use that instead of the native Python logger.

You can access the debug loglevel with `from openage.log import dbg` and use that like you would use the native Python `logging.debug()` function.